src/Compilers/VisualBasic/Portable/Utilities/FunctionExtensions.vb

```diff
@@ -14,6 +14,26 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 'TODO - This is copied from C# and should be moved to common assemble.
 Namespace Microsoft.CodeAnalysis.VisualBasic
     Friend Module FunctionExtensions
+
+#If NETCOREAPP Then
+        <Extension()>
+        Public Function TransitiveClosure(Of T)(relation As Func(Of T, IEnumerable(Of T)), item As T) As HashSet(Of T)
+            Dim closure = PooledObjects.PooledHashSet(Of T).GetInstance
+            Dim stack as New Stack(Of T)()
```

Comment:
why is htis not using pooledstack?

Reply:
Didn't get round to updating it yet. Plus no point if going shared common implementation.